packages/next/src/client/app-link-gc.ts

```diff
@@ -0,0 +1,53 @@
+export function linkGc() {
```

Comment:
This will probably temporarily break the "bench integration tests"

Reply:
How so?